apl-core/src/test/resources/conf/apl-blockchain.properties

```diff
@@ -103,10 +103,10 @@ apl.testDbDir=./apl_test_db
 # Should unit test SQL data be loaded from db/data.sql ?
 apl.testData=true
 
-apl.dbParams=DB_CLOSE_ON_EXIT=FALSE
+apl.dbParams=DB_CLOSE_ON_EXIT=FALSE;AUTO_SERVER=TRUE
 
 # Deprecated,  testnet and mainnet dbs has the same configuration
-apl.testDbParams=DB_CLOSE_ON_EXIT=FALSE;MVCC=TRUE;MV_STORE=FALSE;TRACE_LEVEL_SYSTEM_OUT=3
+apl.testDbParams=DB_CLOSE_ON_EXIT=FALSE;MVCC=TRUE;MV_STORE=FALSE;TRACE_LEVEL_SYSTEM_OUT=3;AUTO_SERVER=TRUE
```

Comment:
This option is deprecated, just remove it.

Reply:
Removed